src/docker/database/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This docker file is used to run the database migrations
+FROM rust:slim-bookworm
+
+ARG RUST_ROOT=src/rust
+ARG DATABASE_URL
+ENV DATABASE_URL=$DATABASE_URL
+WORKDIR $RUST_ROOT/dbv2
+
+COPY . .
```

Comment:
According to the build context from the docker compose, this will copy the entire econia repostory into the image, and it looks like it will try to do this *after* switching the working directory

I think this block should look something more like

```
# Relative to Econia repository root.
ARG RUST_ROOT=src/rust

ARG DATABASE_URL

ENV DATABASE_URL=$DATABASE_URL
COPY $RUST_ROOT .
WORKDIR dbv2
```

Reply:
Per offline discussion, diesel only needs access to the `dbv2` directory tree with the migrations files and the `diesel.toml` config

Additionally it can be somewhat confusing to set `WORKDIR` before the copy operation, because the relevant directory has not yet been copied over

Hence to reduce image caching size and to provide clarity about the working directory, how about?

```Dockerfile
# Relative to Econia repository root.
ARG DATABASE_ROOT=src/rust/dbv2

ARG DATABASE_URL
ENV DATABASE_URL=$DATABASE_URL

COPY $DATABASE_ROOT /db/
WORKDIR /db/
```

Reply:
Addressed in c7683cb9.